lms/djangoapps/courseware/tests/test_view_authentication.py

```diff
@@ -387,6 +403,26 @@ def test_enrollment_period(self):
         self.login(self.global_staff_user)
         self.assertTrue(self.enroll(self.course))
 
+    def test_org_instructor_cannot_access_without_sudo(self):
+        """
+        Test that org instructor cannot load the instructor dashboard without sudo access
+        and it redirect org instructor to sudo password page.
+        """
+        self.login(self.org_instructor_user)
+        url = reverse('instructor_dashboard', kwargs={'course_id': unicode(self.course.id)})
+        response = self.assert_request_status_code(302, url)
+        self.assertIn('sudo/?region=', response['location'])
+
+    def test_org_staff_cannot_access_without_sudo(self):
+        """
+        Test that org staff cannot load the instructor dashboard without sudo access
+        and it redirect org staff to sudo password page.
+        """
+        self.login(self.org_staff_user)
+        url = reverse('instructor_dashboard', kwargs={'course_id': unicode(self.course.id)})
+        response = self.assert_request_status_code(302, url)
+        self.assertIn('sudo/?region=', response['location'])
```

Comment:
(You might be able to do that by having django revers construct the correct url for the redirection.